examples/mnist_deep_avgpool_mpi.py

```diff
@@ -0,0 +1,275 @@
+
```

Comment:
The copyright notice has been cut off here.

Reply:
Fixed this